spec/system/candidate_interface/decoupled_references/review_references_spec.rb

```diff
@@ -115,6 +116,17 @@ def and_i_can_delete_a_reference
     expect(page).not_to have_link 'Delete referee'
   end
 
+  def and_i_can_cancel_a_sent_reference
+    within '#references_sent' do
+      click_link 'Cancel request'
+    end
+    click_button 'Yes I’m sure'
+
+    expect(page).to have_current_path candidate_interface_decoupled_references_review_path
+    expect(page).to have_content("Reference request cancelled for #{@requested_reference.name}")
+    expect(page).to have_content('Cancelled')
```

Comment:
Worth checking the referee received the `reference_cancelled_email`

Reply:
Given it a go. Think it's okay, let me know if not!